### ISSUE TITLE
vagrant ssh sets $TERM inside VM to cygwin instead of xterm-256color

### ISSUE THREAD

**Issue by alankritjoshi** [hashicorp/vagrant#9042]
Please note that the Vagrant issue tracker is reserved for bug reports and
enhancements. For general usage questions, please use the Vagrant mailing list:
https://groups.google.com/forum/#!forum/vagrant-up. Thank you!

### Vagrant version
Vagrant 2.0.0

### Host operating system
Windows 10 Home

### Guest operating system
ubuntu/trusty64

### Vagrantfile
```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "ubuntu/trusty64"
  config.vm.synced_folder '.', '/vagrant', disabled: true
  config.vm.synced_folder "data/", "/vagrant/vagrant_data"
end
```
### Debug output
https://gist.github.com/alankritjoshi/e0adaa682da9575e2d5c2e1bd9c3aeb6

### Expected behavior
``echo $TERM`` inside guest should print xterm-256color

### Actual behavior
``echo $TERM`` inside guest prints cygwin

### Steps to reproduce
Install virtualbox 5.1.28 and vagrant 2.0.0
1. vagrant up
2. vagrant ssh
3. ``echo $TERM``

When I manually SSH into the vm using

``ssh -p <vagrant port> vagrant@127.0.0.1`` 

and then do ``echo $TERM``, I get the expected result as xterm-256color

Am I doing something wrong?

**Comment by chrisroberts** [comment]
Fixed via #9400

**Comment by alankritjoshi** [comment]
Thanks for your efforts!
